bubble_sort.rb

```diff
@@ -0,0 +1,39 @@
+# bubble_sort method
+def bubble_sort(array)
+  len = array.length
+  (0...len - 1).each do |i|
+    (0...len - 1 - i).each do |j|
+      array[j], array[j + 1] = array[j + 1], array[j] if array[j] > array[j + 1]
+    end
+  end
+end
```

Comment:
- [ ] You've got your logic right here but you will need to return the value of the sorted array for your method to be useful. With that said, kindly ensure you return the value of the modified array for your method to work.